RiotNSE/NotificationService.swift

```diff
@@ -479,52 +492,48 @@ class NotificationService: UNNotificationServiceExtension {
                         MXLog.debug("[NotificationService] notificationContentForEvent: Resetting title and body because app protection is set")
                         notificationBody = NSString.localizedUserNotificationString(forKey: "MESSAGE_PROTECTED", arguments: [])
                         notificationTitle = nil
+                        notificationSubTitle = nil
                     }
                     
                     guard notificationBody != nil else {
                         MXLog.debug("[NotificationService] notificationContentForEvent: notificationBody is nil")
                         onComplete(nil)
                         return
                     }
-                    
+
                     let notificationContent = self.notificationContent(withTitle: notificationTitle,
+                                                                       withSubTitle: notificationSubTitle,
                                                                        body: notificationBody,
                                                                        threadIdentifier: threadIdentifier,
                                                                        userId: currentUserId,
                                                                        event: event,
                                                                        pushRule: pushRule,
                                                                        additionalInfo: additionalUserInfo)
-                    
-                    MXLog.debug("[NotificationService] notificationContentForEvent: Calling onComplete.")
-                    onComplete(notificationContent)
+
+                    if #available(iOS 15.0, *) {
+                            self.makeCommunicationNotification(
+                                forEvent: event,
+                                // TODO: use real room/user avatar
+                                senderImage: INImage.systemImageNamed("person.circle.fill"),
```

Comment:
If this was `nil` would it give the full sized app icon until the real avatar is available?

Reply:
Yes, I still did not figure out what one should do there. So if nil or this is the better approach

Reply:
I'd go with `nil` for now as it will look familiar. Having a template image may give the appearance that something has broken.